stories/topics/proc-gcp-additional-configs-update-admin-password.adoc

```diff
@@ -14,5 +14,5 @@ Follow these steps to change the administrator password for both {ControllerName
 . Replace the running {PlatformNameShort} VM instances to use the new administrator password.
 .. Navigate to the *GCP VM Instances* console.
 .. Identify and delete one {ControllerName} VM instance and one {HubName} VM instance for the {PlatformNameShort} deployment.
```

Comment:
It seems that you are only removing one of the two controllers.  Wouldn't this cause one controller with old pwd and one with new?

Reply:
Only have to update the password in the database, so one controller is enough to perform the update.